packages/compiler/package.json

```diff
@@ -5,23 +5,27 @@
   "license": "LGPL",
   "description": "Compiler for Latitude's custom conversation language",
   "type": "module",
-  "main": "src/index.ts",
-  "files": [
-    "src"
-  ],
+  "files": ["dist"],
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
   "scripts": {
     "dev": "rollup -c -w",
+    "build": "rollup -c",
     "test": "vitest run",
     "test:watch": "vitest",
     "prettier": "prettier --write src/**/*.ts",
     "lint": "eslint src",
     "tc": "tsc --noEmit"
   },
-  "peerDependencies": {
-    "acorn": "*",
-    "code-red": "*",
-    "locate-character": "*",
-    "yaml": "*"
+  "dependencies": {
```

Comment:
@latitude-data/compiler is a compiled package so it has to have proper dependencies